gossip/minimal/configpb/config.proto

```diff
@@ -49,6 +49,8 @@ message HubConfig {
   google.protobuf.Duration min_req_interval = 4;
   // Whether the hub is a true Gossip Hub.
   bool is_hub = 5;
+  // Start point to scan a Hub from.
```

Comment:
Can you expand that comment a bit? Given there are multiple hubs it wasn't immediately obvious what it's for. Seemed to me that start index could be per log?

Reply:
Expanded the comment; this covers cases like Crucible, which only has cert-encoded-STHs after entry 350000 -- so we might as well start scanning there.